assets/js/components/dashboard-sharing/DashboardSharingSettingsButton.js

```diff
@@ -85,7 +98,7 @@ export default function DashboardSharingSettingsButton() {
 			<Portal>
 				<Dialog
 					open={ dialogOpen }
-					onClose={ closeDialog }
+					onClose={ () => closeDialog( true ) }
```

Comment:
Same here.

Reply:
Sorted in light of latest review. Thank you @hussain-t!